modules/deploy/83-flowmanager-deploy.yml

```diff
@@ -11,11 +11,13 @@ spec:
     spec:
       containers:
       - name: flow-manager
-        image: gpii/flow-manager
+        image: gpii/universal
         ports:
         - containerPort: 8081
         env:
-        - name: PREFERENCES_SERVER_HOST_ADDRESS
-          value: preferences.default.svc.cluster.local
         - name: NODE_ENV
-          value: gpii.config.cloudBased.production
+          value: gpii.config.cloudBased.flowManager.production
+        - name: GPII_FLOWMANAGER_PREFERENCES_URL
+          value: http://preferences.default.svc.cluster.local/preferences/%userToken
```

Comment:
No, because the preferences service is exposed on port 80 (https://github.com/gpii-ops/gpii-terraform/blob/master/modules/deploy/67-preferences-svc.erb#L15)